build_scripts/build_dependencies_jetson_cuda.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # List of required dependencies
-dependencies=("git-lfs" "libncurses5-dev" "ninja-build" "nasm" "curl" "libudev-dev" "libssl-dev")
+dependencies=("git-lfs" "libncurses5-dev" "ninja-build" "nasm" "curl" "libudev-dev" "libssl-dev" "doxygen" "graphviz")
```

Comment:
@kushaljain-apra 
Do we need graphviz for doxygen?

Reply:
Yes, the graphs that are being generated for each class uses graphviz.

Reply:
Okay